pyof/v0x01/asynchronous/error_msg.py

```diff
@@ -188,7 +188,7 @@ def __init__(self, xid=None, error_type=None, code=None, data=b''):
         super().__init__(xid)
         self.error_type = error_type
         self.code = code
-        self.data = data
+        self.data = BinaryData() if data is None else data
```

Comment:
`BinaryData()` -> `b''`; it's bad having GenericTypes when they are not needed.
The cleaner way is to set `b''` directly in the `__init__` method - same behavior, better readability.